rplugin/python3/denite/source/grep.py

```diff
@@ -41,6 +41,16 @@ def on_close(self, context):
             self.__proc.kill()
             self.__proc = None
 
+    def highlight_syntax(self):
+        input_str = self.context['__input']
+        self.vim.command(
```

Comment:
I deliberately put highlight_syntax here because different source will have different highlight schema. If we define highlight in UI, source highlighting will not be customizable.